### PR TITLE
Soap Improvements

### PR DESCRIPTION
**Pull request by DEATHB4DEFEAT**
# Media

(You have to sprint to slip, and the red dots are from the soap bomb)

https://github.com/Simple-Station/Parkstation-Friendly-Chainsaw/assets/77995199/9038d5aa-222e-4fb1-b33c-a863d7df6928

# Changelog

:cl:
- tweak: Soap is smaller, harder to slip on, and stuns less

### CODE REVIEW

**Comment by LovelyLophi** [comment]
I feel this only serves to make playing the game less fun, slipping security with soap is a tried and true passtime while also allowing me to taze Greyshirts.

**Comment by Pspritechologist** [comment]
My goal is not for Space Station to lack fun, or to be boring, but rather for it to be the straight-man universe required in order to allow shenanigans to hold water. I think GTA vs Saints Row is a good comparison.

One complaint often thrown against Saint's Row 3 is the world's complete jump off the deep end in terms of sanity. There are giant naked brutes running through the streets, a literal murder gang is treated like a pop band, and zombies are real. This complete lack of straight man makes things like running around with a giant dildo baseball bat seem commonplace, and causes them to lose their absurdity. GTA on the other hand is filled with quite normal people just leading their lives, which makes all of the terrible things you can do in it *seem* terrible. Soap is a commonplace item, it's something every human should be able to find without even really thinking about it. It's normal, basic, and although watching people go careening through the halls after meandering onto a cubic meter of it might be funny, it also makes the world seem like it's *supposed* to be slapstick. Space lubricant on the other hand *isn't* any of those things, it's an absurd item with an absurd concept. It's inherently silly, and therefore the silly effects of it fit. No one should bat an eye at someone carrying soap, but an industrial tank of lube? That's strange.

Making the world a little more normal and a little more sane is not with the purpose of making it less fun. It's with the purpose of making the *world* less slapstick, so the events that take place in it can be more fun.

**Comment by LovelyLophi** [comment]
> Making the world a little more normal and a little more sane is not with the purpose of making it less fun. It's with the purpose of making the _world_ less slapstick, so the events that take place in it can be more fun.

I have not heard one person talking about the realism of slipping on soap, if you believe it truly is that horrible for soap to be slippery then I suppose you can nerf it.

File: Resources/Prototypes/Entities/Objects/Specific/Janitorial/soap.yml [hunk 1]
```diff
@@ -221,7 +223,7 @@
   - type: SolutionContainerVisuals
     fillBaseName: omega-
   - type: Slippery
-    paralyzeTime: 7
+    paralyzeTime: 5 # Parkstation-BetterSoap
```
Comment (Pspritechologist) [inline Resources/Prototypes/Entities/Objects/Specific/Janitorial/soap.yml:226]: This seems like a silly change. The amount of time is still annoyingly long, so what's the difference between the annoyingly long it was and the annoyingly long it is now?

File: Resources/Prototypes/Entities/Objects/Specific/Janitorial/soap.yml [hunk 2]
```diff
@@ -154,8 +156,8 @@
     layers:
     - state: syndie-soaplet
   - type: Slippery
-    paralyzeTime: 5
-    launchForwardsMultiplier: 2.5
+    paralyzeTime: 2 # Parkstation-BetterSoap
+    launchForwardsMultiplier: 1 # Parkstation-BetterSoap
```
Comment (Pspritechologist) [inline Resources/Prototypes/Entities/Objects/Specific/Janitorial/soap.yml:160]: This seems like a strong nerf for a bought item.

**Comment by Pspritechologist** [comment]
Before/After media?

**Comment by LovelyLophi** [comment]
I feel that the syndicate and omega soap should always slip (similarly to space lube) and the stuntimes should be more than enough time to cuff.

**Comment by Pspritechologist** [comment]
> I feel that the syndicate and omega soap should always slip (similarly to space lube) and the stuntimes should be more than enough time to cuff.

@DEATHB4DEFEAT ?

**Comment by DEATHB4DEFEAT** [comment]
> I feel that the syndicate and omega soap should always slip (similarly to space lube) and the stuntimes should be more than enough time to cuff.

No, and the Syndicate soap *might* be long enough to cuff but the soaplets definitely are

**Pull request merged**